test/testdata/generate.sh

```diff
@@ -58,58 +48,55 @@ scaffold_test_project() {
     header_text 'Creating APIs ...'
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --force
-    $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
```

Comment:
if you remove it from here then it will no longer validate the --force in line 53.
So, it needs to be revert